parse_csv.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+
+def strip_prefix(input_string, prefix):
+    if input_string.startswith(prefix):
+        return input_string[len(prefix):]
+    return input_string
+
+def add_to_output(export_output, row, prefix):
```

Comment:
so, if i understand it correctly this function normalizes the url via "strip_prefix" et.al. and then sums the numerical fields and concatenates strings with a slash. it also seems you don't have any good way to figure out if the value is a string or an integer and so you are relying on value error, hmmm

here is the first pass of unravelling i'd do; i'd also prolly follow it up with another 3 passes till the whole script is 30 lines long

```python
def sum_dicts(by_url, row, prefix):
    url = "https://www.parse.ly" + strip_prefix(row['URL'], prefix)

    rec = by_url.get(url, {})
    for key, val in row.items():
        if not rec.get(key):
            rec[key] = val
        else:
            # if we have at least one value, we can try determining if it's a float or a string
            try:
                rec[key] = str(float(rec[key] or 0) + float(val or 0))
            except ValueError:
                # assuming string and just concatenating with a padded " / "
                values = set((rec[key]  or "").split(" / "))
                values.add(val)
                rec[key] = " / ".join(sorted([val for val in values if val]))
        else:
            # filling blanks is fine too
            rec[key] = val

    return url, rec
```